Sources/Yams/Parser.swift

```diff
@@ -287,9 +287,8 @@ private class Event {
         return Node.Scalar.Style(rawValue: event.data.scalar.style.rawValue)!
     }
     var scalarTag: String? {
```

Comment:
This can return a `String` now

Reply:
It can't since `string(from:)` still returns `String?`